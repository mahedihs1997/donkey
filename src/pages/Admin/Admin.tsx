import { Typography } from '@material-ui/core';
import React from 'react';

const Admin = () => {
  return (
    <div>
      <Typography variant="body1">Admin Dashboard</Typography>
    </div>
  );
};

Admin.getInitialProps = async () => {
  return {
    showLayout: true,
    title: 'Admin',
  };
};

export default Admin;
