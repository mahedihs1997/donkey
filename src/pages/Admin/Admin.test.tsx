import React from 'react';
import TestRenderer from 'react-test-renderer';

import Admin from './Admin';

describe('<Admin />', () => {
  it('should render without throwing an error and match snapshot', () => {
    const testRenderer = TestRenderer.create(<Admin />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});
