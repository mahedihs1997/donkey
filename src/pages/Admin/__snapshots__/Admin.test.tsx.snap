// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Admin /> should render without throwing an error and match snapshot 1`] = `
<div>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Admin Dashboard
  </p>
</div>
`;
